tests/lib/rules/preserve-caught-error.js

```diff
@@ -722,5 +728,58 @@ ruleTester.run("preserve-caught-error", rule, {
 				},
 			],
 		},
+		/* 25. When multiple `cause` properties are present. */
+		{
+			code: `try {} catch (error) {
+				throw new Error("Something failed", { cause: error, cause: anotherError });
+			}`,
+			errors: [{ messageId: "incorrectCause" }],
+		},
```

Comment:
This was `valid` before this change, so I've changed the tag to `feat` because this change can produce more lint errors.